File: swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/mailbaby-client-csharp-dotnet2/Model/MailBlocks.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// The complete set of blocked addresses and message patterns returned by &#x60;GET /mail/blocks&#x60;.  Three independent block sources are combined into one response. Use &#x60;POST /mail/blocks/delete&#x60; with a &#x60;from&#x60; address from any entry to delist it.
  /// </summary>
  [DataContract]
  public class MailBlocks {
    /// <summary>
    /// Messages flagged by the `LOCAL_BL_RCPT` rspamd rule in the last 5 days. These are messages sent to recipients on a local block list.
    /// </summary>
    /// <value>Messages flagged by the `LOCAL_BL_RCPT` rspamd rule in the last 5 days. These are messages sent to recipients on a local block list.</value>
    [DataMember(Name="local", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "local")]
    public List<MailBlockClickHouse> Local { get; set; }

    /// <summary>
    /// Messages flagged by the `MBTRAP` rspamd rule in the last 5 days. These triggered MailBaby's honeypot / trap address detection.
    /// </summary>
    /// <value>Messages flagged by the `MBTRAP` rspamd rule in the last 5 days. These triggered MailBaby's honeypot / trap address detection.</value>
    [DataMember(Name="mbtrap", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "mbtrap")]
    public List<MailBlockClickHouse> Mbtrap { get; set; }

    /// <summary>
    /// Senders whose messages contained spam-indicative subjects (containing `@`, `smtp`, `socks4`, or `socks5`) with more than 4 occurrences of the same subject in the last 3 days.
    /// </summary>
    /// <value>Senders whose messages contained spam-indicative subjects (containing `@`, `smtp`, `socks4`, or `socks5`) with more than 4 occurrences of the same subject in the last 3 days.</value>
    [DataMember(Name="subject", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "subject")]
    public List<MailBlockRspamd> Subject { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MailBlocks {\n");
      sb.Append("  Local: ").Append(Local).Append("\n");
      sb.Append("  Mbtrap: ").Append(Mbtrap).Append("\n");
      sb.Append("  Subject: ").Append(Subject).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/MailBlocks.scala

package org.openapitools.client.model


case class MailBlocks (
    /* Messages flagged by the `LOCAL_BL_RCPT` rspamd rule in the last 5 days. These are messages sent to recipients on a local block list. */
    _local: List[MailBlockClickHouse],
    /* Messages flagged by the `MBTRAP` rspamd rule in the last 5 days. These triggered MailBaby's honeypot / trap address detection. */
    _mbtrap: List[MailBlockClickHouse],
    /* Senders whose messages contained spam-indicative subjects (containing `@`, `smtp`, `socks4`, or `socks5`) with more than 4 occurrences of the same subject in the last 3 days. */
    _subject: List[MailBlockRspamd]
)
object MailBlocks {
    def toStringBody(var_local: Object, var_mbtrap: Object, var_subject: Object) =
        s"""
        | {
        | "local":$var_local,"mbtrap":$var_mbtrap,"subject":$var_subject
        | }
        """.stripMargin
}
